app/server/utils.py

```diff
@@ -391,7 +391,8 @@ def parse(self, file):
                 yield data
                 data = []
             try:
-                j = json.loads(line)
+                #j = json.loads(line)
+                j  = json.loads(line.decode('utf-8'))
```

Comment:
Instead of calling `.decode` here, it may be better to wrap the entire file in a `TextIOWrapper` which will handle the encoding and ensure that we read the buffer correctly line by line. For an example, take a look at the [PlainTextParser implementation](https://github.com/chakki-works/doccano/blob/beeba225c7c916b3068337c244fafe487a356378/app/server/utils.py#L343).